docs/api/cloudlink/packets/delete_post.md

```diff
@@ -0,0 +1,11 @@
+# `delete_post`
+
+This packet is sent when a is deleted, when a client gets this packet, it should search any local cache for the post_id and remove any matching posts.
```

Comment:
```suggestion
This packet is sent when a post is deleted. 

```

generally, these docs are intended to be a reference and shouldn't tell clients what to do, that's what the intent of the apic examples and stuff are for